src/python/pants/source/wrapped_globs.py

```diff
@@ -91,6 +91,11 @@ def files(self):
   def files_hash(self):
     return self._files_hash
 
+  def iter_relative_paths(self):
```

Comment:
Better to call this `paths_from_buildroot_iter()` (and below).

`relative` is a vague term.